KrysiaBartek2024/source/toc_entries.tex

```diff
@@ -0,0 +1,1105 @@
+\section{\texorpdfstring{1\mins\ opening}{1mOpening}}\label{sec:1mOpening}
+
+\subsubsection*{1\clubs\ -- ?}
+\begin{itemize}
+    \item 1\diams\ = 0-6
+    \item 1\hearts\ = 4+\hearts
+    \item 1\spades\ = 4+\spades
+    \item 1\ntx\ = 7-10, no 4\majs
+    \item 2\clubs\ = 12-14 BAL or \clubs, \gf
+    \item 2\diams\ = no 4\majs, 5+\diams, \gf
```

Comment:
czemu no 4 majs

Reply:
lol jak mam 4M to zawsze pokazywałam do tej pory xd